CONTRIBUTORS.md

```diff
@@ -16,6 +16,10 @@ Role: Web Developer
 GitHub: [zachgarwood](https://github.com/zachgarwood)
 GitLab: [zachgarwood](https://gitlab.com/zachgarwood)
 
+### Trevin Alonzo
+Role: Web Developer
+Github: [web-dev-trev]https://github.com/web-dev-trev
```

Comment:
For markdown links, the url needs to be inside parens:
`[web-dev-trev](https://github.com/web-dev-trev)`